spec/controllers/email_address_controller_spec.rb

```diff
@@ -0,0 +1,116 @@
+require "rails_helper"
+
+RSpec.describe EmailAddressController, type: :controller do
+  describe "GET #edit" do
+    it "renders the edit template with a new EmailAddressForm" do
+      get :edit
+
+      expect(assigns(:form)).to be_a(EmailAddressForm)
+      expect(response).to render_template(:edit)
+    end
+  end
+
+  describe "POST #update" do
+    let(:valid_email_address) { "test@example.com" }
+    let(:mixed_case_email_address) { "Test@Example.COM" }
+    let(:invalid_email_address) { "" }
+
+    context "when the form is valid" do
+      context "and an archived intake exists with the email address" do
+        let!(:archived_intake) { create :state_file_archived_intake, email_address: valid_email_address }
+        # TODO update this test with logging and the proper redirects
+        it "creates a request, and redirects to the root path" do
+          post :update, params: {
+            email_address_form: { email_address: valid_email_address }
+          }
+          expect(assigns(:form)).to be_valid
+          active_archived_intake = controller.send(:current_archived_intake)
+          expect(active_archived_intake.email_address).to eq(valid_email_address)
+          expect(active_archived_intake.hashed_ssn).to eq(archived_intake.hashed_ssn)
+          expect(active_archived_intake.id).to eq(archived_intake.id)
+
+          expect(response).to redirect_to(
+                                root_path
+                              )
+        end
+
+        it "matches email case insensitively" do
+          post :update, params: {
+            email_address_form: { email_address: mixed_case_email_address }
+          }
+
+          expect(assigns(:form)).to be_valid
+
+          active_archived_intake = controller.send(:current_archived_intake)
+          expect(active_archived_intake.email_address).to eq(valid_email_address)
+          expect(active_archived_intake.hashed_ssn).to eq(archived_intake.hashed_ssn)
+          expect(active_archived_intake.id).to eq(archived_intake.id)
+
+          expect(response).to redirect_to(
+                                root_path
+                              )
+        end
+
+        it "resets verification session variables sets the email address" do
```

Comment:
what is this testing that the first test in this block isn’t testing?

Reply:
can’t you add these expectations to the first test?

Reply:
good point